pkg/operator/ceph/object/status.go

```diff
@@ -69,7 +69,10 @@ func updateStatusBucket(client client.Client, name types.NamespacedName, phase c
 		logger.Warningf("failed to retrieve object store %q to update status to %v. %v", name, phase, err)
 		return
 	}
-
+	if objectStore.Status == nil {
```

Comment:
If the status isn't set yet, let's initialize it so it can be set for the first time.

Reply:
@travisn initialize and continue or initialize and return?

Reply:
Initialize and continue. The struct just needs to be created the first time so the fields can be set.

Reply:
Thanks for the suggestion.updated now PTAL